.travis.yml

```diff
@@ -1,10 +1,24 @@
 language: dart
+
 dart:
-  - stable
+  - 2.0.0
+  - dev
+
 install:
   - gem install coveralls-lcov
+
 script: ./tool/travis.sh
+
 after_success:
   - coveralls-lcov var/lcov.info
+
 env:
   - DARTANALYZER_FLAGS=--fatal-warnings
+
+# Only building master means that we don't run two builds for each pull request.
```

Comment:
Change this to document what the other build is that we're avoiding -- I'd assume it's not because of the `dev` Dart SDK, but something else?